dmutils/filters.py

```diff
@@ -66,3 +66,27 @@ def capitalize_first(maybe_text):
         return [capitalize_first(item) for item in maybe_text]
 
     return maybe_text
+
+# find repeated sequences of '\r\n\', optionally separated by a space character
+_multiple_newlines_re = re.compile(r'(\r\n *){2,}')
+# find \r\n sequences
+_single_newline_re = re.compile(r'(\r\n)')
+
+
+@evalcontextfilter
+def preserve_line_breaks(eval_ctx, value, question_type):
+    if question_type and question_type != 'textbox_large':
+        return value
+
+    # Turn potential markdown objects into string
+    value = '{}'.format(escape(value))
```

Comment:
I think this needs `u'{}'`, since original value is still unicode. It's also worth writing a test with unicode characters for this